kano_greeter/user_list.py

```diff
@@ -70,32 +75,19 @@ def add_account(*_):
 
 
 
-class User(Gtk.EventBox):
+class User(KanoButton):
     HEIGHT = 50
 
     def __init__(self, username):
-        Gtk.EventBox.__init__(self)
+        KanoButton.__init__(self, text=username.title(), color="orange")
```

Comment:
We may not be very good at it but let's try to be consistent with `'` and `"`. Nearly all of the current code uses `'` so we'll keep that as the preference here.

At some point we should also change the couple of other cases where `"` is used.